src/classes/UTIL_Describe.cls

```diff
@@ -269,6 +269,33 @@ public class UTIL_Describe {
         return fieldMap;
     }
 
+    /*******************************************************************************************************
+     * @description Returns a comma delimited string with all accessible fields of an object, used for queries.
+     * @param sobjectname Name of the object to retrieve field list for.
+     * @return String A comma delimited String with all fields of an object.
+     */
+    public static String getAllFieldsQueryStringFromObject(String sObjectName) {
```

Comment:
I have some potential reservations about adding this method to the Describe library. I realize it's here because there's a similar method in GEM and we want to utilize similar setup for the new unit test to verify that we aren't failing in the equivalent test that was failing in GEM. However, it appears that its purpose is to support a "SELECT  *" type functionality, and that's not considered a great practice in Salesforce development.  

I realize that we are doing this essentially already both in GEM and in parts of BDI, but I might prefer to avoid adding a method that's so clearly designed to support that practice -  one option is that we could simply bring this code inline within the unit test, although we also might want to consider grabbing the globally available  "listStrDataImportFields" property of the BDI_DataImportService class and doing a String.join() of those results as well?

Reply:
Didn't know listStrDataImportFields already existed...